src/components/Favicon/Auto/hooks/useFaviconGrabbers.js

```diff
@@ -0,0 +1,55 @@
+import UrlService from '../../../../services/UrlService';
+import useCorsProxy from './useCorsProxy';
+
+const getFaviconFromWebsite = (url, faviconFileName) => {
```

Comment:
All functions inside this file do not look like React Hook. Why do we use `hook` as a part of the file name?

Reply:
I thought it would be a good idea to combine all of the functions into one object to be accessible with something like `const { getFaviconFromWebsite } = useFaviconGrabbers()`.
But yeah, there is no reason to call these functions as hooks. Moved everything to `FaviconGrabberService` class.